data/scenarios/Testing/201-require/533-reprogram-simple.yaml

```diff
@@ -5,7 +5,9 @@ description: |
   the target robot doesn't have.
   https://github.com/swarm-game/swarm/pulls/533
 objectives:
-  - condition: |
+  - goal:
+      - Reprogram
```

Comment:
Adding "fake" objective descriptions seems like an opportunity to better document the test suite.

Reply:
Exactly, I sometimes inspect the tests from main menu and having a goal would be helpful.